gen2-crestereo-stereo-matching/requirements.txt

```diff
@@ -1,4 +1,4 @@
-opencv-python==4.5.1.48
-depthai==2.15.2
+opencv-python==4.5.5.62
```

Comment:
Is this specifically needed for this demo? If so, please add a comment that this version is required